File: Data Structures & Algorithms/largest-3-same-digit-number-in-string/submission-2.py
class Solution:
    def largestGoodInteger(self, num: str) -> str:
        nums = list(num)
        highest = -1
        for i in range(1, len(num)-1):
            if nums[i] == nums[i-1] == nums[i+1] and int(nums[i]) > highest:
                highest = int(nums[i])
        
        if (highest < 0): return ""
        elif (highest == 0): return "000"
        else: return str(highest * 111)

File: Data Structures & Algorithms/concatenation-of-array/submission-2.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
        vector<int> doubled = nums;
        for (int i = 0; i < nums.size(); i++) {
            doubled.push_back(nums[i]);
        }
        return doubled;
    }
};